Setup/UpgradeSchema.php

```diff
@@ -128,4 +130,43 @@ private function setupFeatureSwitchTable(SchemaSetupInterface $setup) {
         )->setComment("Bolt feature switch table");
           $setup->getConnection()->createTable($table);
     }
+
+    private function setupLogTable($setup){
+        $tableCreated = $setup->getConnection()->isTableExists('bolt_log');
+        if ($tableCreated) {
+            return;
+        }
+
+        $table = $setup->getConnection()
+            ->newTable($setup->getTable('bolt_log'))
+            ->addColumn(
+                'id',
+                \Magento\Framework\DB\Ddl\Table::TYPE_INTEGER,
+                null,
+                ['identity' => true, 'unsigned' => true, 'nullable' => false, 'primary' => true],
+                'ID'
+            )
+            ->addColumn(
+                'transaction_id',
+                \Magento\Framework\DB\Ddl\Table::TYPE_TEXT,
```

Comment:
why text? is this what transaction ids in M2 are?

Reply:
@roopakv It’s the Bolt transaction ID, so we need to use the text or varchar type. FYI after the schema has run, the Mysql type will be varchar. See here - https://prnt.sc/r4m658